src/components/Installer/index.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState } from "react";
+import styles from "./styles.module.css";
+
+export default function CondaInstallerSelector() {
+  const [os, setOs] = useState("");
+  const [arch, setArch] = useState("");
+  const [installerType, setInstallerType] = useState("");
+
+  const osArchitectures = {
+    Linux: ["x64", "aarch64", "ppc64le"],
+    macOS: ["x64", "ARM64"],
+    Windows: ["x64", "ARM64 (Beta)"],
+  };
+
+  const installerData = {
+    Anaconda: {
+      name: "Anaconda",
+      url: "https://docs.anaconda.com/free/anaconda/install/index.html",
+      defaultChannel: "defaults",
+      description: "Complete distribution for scientific Python development",
+    },
+    Miniconda: {
+      name: "Miniconda",
+      url: "https://docs.conda.io/en/latest/miniconda.html",
```

Comment:
```suggestion
      url: "https://docs.anaconda.com/miniconda/index.html",
```
The Miniconda docs have also been moved from the conda docs to the Anaconda docs.